### DISCUSSION TITLE
Does somebody receive digital badge ? And why all issues still with tag under review?

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/hack-together-dotnet#115] [Q&A]
Does somebody receive a digital badge? And why are all issues still with tags under review?
I would like to close access to my repository, can I do it? Or Microsoft still reviewing projects?
@aaronpowell @justinyoo @codemillmatt @edilma @msftgits @daisychaussee
@microsoftopensource @jmc9000 @edilma @heathbm @eltociear

**Comment by codemillmatt** [comment]
Hi @YuriyMorozyuk95 - the digital badges will be coming soon. And I've removed the labels from the issues. You can see the winners here: https://devblogs.microsoft.com/dotnet/great-dotnet-8-hack-winners/
